### PR TITLE
Fix deadman notification text not being expandable and clean up code (EXPOSUREAPP-4251)

### PR DESCRIPTION
**Pull request by d4rken**
Setting the style to `BigTextStyle` to make the notification expandable and refactor duplicate code away.

## Testing
View the deadman notification on a device with a small screen, try to expand it, and click it.

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt [hunk 1]
```diff
@@ -37,6 +38,12 @@ class AndroidModule {
         @AppContext context: Context
     ): NotificationManagerCompat = NotificationManagerCompat.from(context)
 
+    @Provides
+    @Singleton
+    fun notificationManager(
+        @AppContext context: Context
+    ): NotificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt:46]: There is a handy extension for that in `core-ktx`
`context.getSystemService< NotificationManager>`

**Review by mtwalli — DISMISSED** [review]
LGTM

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt (hunk 1 above)
Comment (ralfgehrer) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt:47]: I preferred the original solution without the `!!`

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/AndroidModule.kt:47]: I understand, but it has the same result at the end. `getSystemService` returns a nullable `NotificationManager`casting it to a NonNullable type would throw a null exception pointer too.

**Review by mtwalli — APPROVED** [review]

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**